docs/Researcher/user-interface/workspaces/create/workspace-v2.md

```diff
@@ -1,40 +1,80 @@
 # Create a new workspace
 A Workspace is assigned to a project and is affected by the project’s quota just like any other workload. A workspace is shared with all project members for collaboration.
 
-To create a new workspace:
+!!! Note
+      * You must have at least one project configured in the system. To configure a project, see [Creating a project](../../../../admin/admin-ui-setup/project-setup.md#create-a-project).
+      * You must have at least 1 researcher assigned to the project.
 
-1. Press **Workspaces** on left menu, then press *New workspace*.
-2. Select a project from the project tiles. If your project is not listed, use the *Search projects* box to find a project.
+Use the *Jobs form* below if you have not enabled the *Workloads* feature.
 
-    !!! Note
-        * You must have at least one project configured in the system. To configure a project, see [Creating a project](../../../../admin/admin-ui-setup/project-setup.md#create-a-project).
-        * You must have at least 1 researcher assigned to the project.
+=== "Jobs enabled"
+    To create a new workspace:
 
-3. Select a template from the template tiles. If your template is not listed, use the *Search templates* box to find a template. Choose *Start from scratch* if you do not have, or want to use a template.
+      1. Press **Workspaces** on left menu, then press *New workspace*.
+      2. Select a project from the project tiles. If your project is not listed, use the *Search projects* box to find a project.
+      3. Select a template from the template tiles. If your template is not listed, use the *Search templates* box to find a template. Choose *Start from scratch* if you do not have, or want to use a template.
 
-     A template contains a set of predefined building blocks as well as additional configurations which allow the user to immediately create a templated-based workspace.
+          A template contains a set of predefined building blocks as well as additional configurations which allow the user to immediately create a templated-based workspace.
 
-4. Enter a name for your workspace and press *Continue*.
-5. Select an environment from the tiles. If your environment is not listed, use the *Search environments* box to find it. Press *New environment* to create an environment if needed.
+      4. Enter a name for your workspace and press *Continue*.
+      5. Select an environment from the tiles. If your environment is not listed, use the *Search environments* box to find it or press [*New environment*](workspaces/create/create-env.md) to create a new environment. Press  to create an environment if needed. In the *Set the connection for your tool(s)*, enter the URL of the tool if a custom URL has been enabled in the selected environment. Use the *Private* toggle to lock access to the tool to only the creator of the environment.
+ 
```

Comment:
@jasonnovichRunAI this is incorrect: "Use the *Private* toggle to lock access to the tool to only the creator of the environment."

this is correct: "Use the *Private* toggle to lock access to the tool to only the creator of the workload"